apps/prairielearn/src/lib/code-caller/code-caller-native.ts

```diff
@@ -342,16 +337,17 @@ export class CodeCallerNative {
       env.DROP_PRIVILEGES = '1';
     }
 
-    /** @type {import('child_process').SpawnOptions} */
-    const options = {
+    const options: SpawnOptions = {
       cwd: import.meta.dirname,
       // stdin, stdout, stderr, data, and restart confirmations
-      stdio: ['pipe', 'pipe', 'pipe', 'pipe', 'pipe'],
+      stdio: ['pipe', 'pipe', 'pipe', 'pipe', 'pipe'] as IOType[],
       env,
     };
-    const child = /** @type {CodeCallerNativeChildProcess} */ (
-      child_process.spawn(cmd, args, options)
-    );
+    const child: CodeCallerNativeChildProcess = child_process.spawn(
```

Comment:
The `as ...` should be sufficient.

```suggestion
    const child = child_process.spawn(
```